src/routes/Products.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+import { useFetchProducts } from "../store/products/products.hooks";
+import { useEffect } from "react";
+import { Link } from "react-router-dom";
+
+const Products: React.FC = () => {
+  const { submit: fetchProducts, data = [], error } = useFetchProducts();
+
+  useEffect(() => {
+    fetchProducts();
+  }, []);
+
+  return (
+    <div>
+      <ul>
+        {data.map(product => (
+          <li key={`products-product-${product.product_id}`}>{product.name}</li>
```

Comment:
I haven't seen that `${}` before. But I can see we are here referring to the JSON from the endpoint and displaying the list of products.

Reply:
Useful
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals